### PR TITLE
Some general cleanup and updates

### PR DESCRIPTION
**Pull request by parlough**

### CODE REVIEW

File: source/plugin/workspace/index.rst [hunk 1]
```diff
@@ -13,3 +17,10 @@ Contents
 
     idea
     eclipse
+
+.. _IntelliJ: https://www.jetbrains.com/idea/
+.. _Eclipse: https://www.eclipse.org/
+.. _Netbeans: https://netbeans.org/
+.. _`VS Code`: https://code.visualstudio.com/
+.. _Atom: https://atom.io/
+.. _`Sublime Text`: https://www.sublimetext.com/
```
Comment (ST-DDT) [inline source/plugin/workspace/index.rst]: Newline at EOF

File: source/preparing/text.rst [hunk 2]
```diff
@@ -9,6 +9,18 @@ a different text editor may prove to be useful.
 Downloads
 =========
 
+Visual Studio Code
```
Comment (ST-DDT) [inline source/preparing/text.rst:12]: Why did you move this section to the top?

Comment (parlough) [reply source/preparing/text.rst:12]: As VS Code has proven to be the more capable editor, with a larger userbase, and a more guaranteed future. Also it seems in the community, it is the more common choice. It's a bit opinionated but I do believe it will provide better development experience.

File: source/about/faq.rst [hunk 3]
```diff
@@ -12,22 +12,22 @@ For Everyone
...
-:doc:`../server/getting-started/jre/`
+the server on too, besides that nothing is required. Learn more about choosing and installing the correct Java version
+here: :doc:`../server/getting-started/jre/`
 
 Where do I get Sponge?
 ----------------------
 
-Beta development builds of SpongeForge and SpongeVanilla are already available:
-`Sponge Downloads <https://spongepowered.org/downloads>`_
+You can download both stable and experimental builds of SpongeForge and SpongeVanilla on our
```
Comment (phit) [inline source/about/faq.rst]: `You can download SpongeForge and SpongeVanilla on our..`

wanna get away from that stupid stable/experimental naming, don't add it to the docs pls

Comment (parlough) [reply source/about/faq.rst]: Sure.

File: source/about/faq.rst [hunk 4]
```diff
@@ -60,8 +67,10 @@ history page: :doc:`history`
 Can I run Bukkit Plugins with this?
 -----------------------------------
 
-Sadly no, not natively. Sponge is using its own API (Sponge API), while Bukkit is using the Bukkit API. However there is the
-possibility that a third-party plugin for Sponge implements the Bukkit API on top of Sponge API.
+Sadly no, not natively. Sponge is using its own API (Sponge API), while Bukkit is using the Bukkit API. While third parties
```
Comment (phit) [inline source/about/faq.rst]: `Sadly no, not natively.` remove? I don't think anyone here is disappointed about that, nor do we think a plugin that adds bukkit comp is a good idea

File: source/about/introduction.rst [hunk 5]
```diff
@@ -37,8 +37,8 @@ common functionality of mixins.
 Where Can I Download Sponge?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Beta development builds of SpongeForge and SpongeVanilla can be found here:
-`Sponge Downloads <https://spongepowered.org/downloads>`_
+You can download both stable and experimental builds of SpongeForge and SpongeVanilla on our
```
Comment (phit) [inline source/about/introduction.rst]: same as above

**Review by ST-DDT — APPROVED** [review]

**Review by Inscrutable — APPROVED** [review]
Lantern added, excised mentions of Pore,
and our Sponge API isn't new any more.
Meronat's new broom brings a clean sweep,
This new apprentice looks like one we can keep.
